### ISSUE TITLE
Guide to discovering articles for PJC

### ISSUE THREAD

**Issue by chartgerink** [SamanthaHindle/preprint_JournalClub#11]
This is a great idea, but for people starting with Preprints and seeing the value of reviewing them, it might be a good idea to provide some outline on how to discover preprints? Considering there are several repositories, some have already been published (defeating the PJC idea), it is important to have the right discovery process to find preprints relevant to the JC's interests.

It could be so simple as providing a resource list of preprint places, and as granular as curating several lists for various topics.

Maybe we can discuss here?

**Comment by SamanthaHindle** [comment]
This is a great idea @chartgerink! Thank you so much for opening this issue. Hopefully once the Central Service that ASAPbio are initiating is launched next year, it will be easier to discover preprints but until then this list will be a wonderful resource. Thanks!

**Comment by dasaderi** [comment]
Great idea @chartgerink! Thank you!

**Comment by chartgerink** [comment]
So I found out about this application developed to discover preprints. It might be worthwhile checking out for discovery purposes: https://jhubiostatistics.shinyapps.io/papr/

I am looking whether I can contribute a database of OSF preprints into this so it is useful outside of biological sciences, but I think it's already usefule for ASAPbio (includes all bioRxiv abstracts)

**Comment by dasaderi** [comment]
@chartgerink thank you! I'm a bit unclear on the goal of the app. It says in for "rating" preprints more than finding them, but I guess the two things are connected.

**Comment by SamanthaHindle** [comment]
Hi @chartgerink We have now developed a [doc about preprints(https://github.com/SamanthaHindle/preprint_JournalClub/blob/master/Starter_Kit/PPJC_Info_Doc.md) which includes a list of places to find preprints. If you have time. would you mind taking a look and let us know if it's thorough enough. Thanks for your help and suggestions. Best, Sam

**Comment by chartgerink** [comment]
@SamanthaHindle great! Maybe the SHARE preprint aggregate database might be worthwhile to add. (https://share.osf.io/discover?type=preprint). I figure you're only focusing on Bio? Too bad, because there are more and this principle can be extended and then funneled down into subgroups. :)

**Comment by dasaderi** [comment]
Thank you @chartgerink for you comment. I think you are right, it would be good to expand to other fields, but bio is what we know best (and frankly what is most behind in terms of open). I think we can model (and test) the project around bio but make it so that other fields can do the same.

**Comment by chartgerink** [comment]
Yes of course :) I want to do a preprint journal club for psych, because there are many cool papers coming out on PsyArxiv (amongst others). Will build on this and contribute as I go. That's the beauty of open :)

**Comment by beastlibrarian** [comment]
Not sure if this is helpful: https://cshl.libguides.com/c.php?g=653424&p=4584615
Above guide includes link to journal preprint policies: https://en.wikipedia.org/wiki/List_of_academic_journals_by_preprint_policy
And includes some examples of preprint servers.

**Comment by SamanthaHindle** [comment]
Hi @beastlibrarian 
These links are really helpful! Thank you! I've added to our doc about preprints.

Thanks so much! ❤️
